bridge/steam/steam.go

```diff
@@ -73,22 +72,20 @@ func (b *Bsteam) Send(msg config.Message) (string, error) {
 		for _, rmsg := range helper.HandleExtra(&msg, b.General) {
 			b.c.Social.SendMessage(id, steamlang.EChatEntryType_ChatMsg, rmsg.Username+rmsg.Text)
 		}
-		if len(msg.Extra["file"]) > 0 {
-			for _, f := range msg.Extra["file"] {
-				fi := f.(config.FileInfo)
+		for _, f := range msg.Extra["file"] {
+			fi := f.(config.FileInfo)
+			if fi.Comment != "" {
```

Comment:
The logic here would be way more readable with:
```go
msgContent := []string{msg.Username, msg.Text}
if fi.Comment != "" {
	msgContent = append(msgContent, fi.Comment)
}
if fi.URL != "" {
	msgContent = append(msgContent, fi.URL)
}
b.c.Social.SendMessage(id, steamlang.EChatEntryType_ChatMsg, strings.Join(msgContent, ": "))
```

Reply:
sorry disagree here, it could maybe be more readable, but I don't think your suggestion is more readable.